.circleci/config.yml

```diff
@@ -43,18 +45,29 @@ commands:
           name: Start StarWars Server
           background: true
       - run:
-          command: cd SimpleUploadServer && npm install && npm start
+          command: cd SimpleUploadServer && nvm use && npm install && npm start
```

Comment:
Required to tell nvm to actually use the `.nvmrc` file created to lock SimpleUploadServer to node v12.